www/utils/swrCache.ts

```diff
@@ -0,0 +1,12 @@
+import type { Cache } from 'swr';
+
+const memoryCache = new Map<string, any>();
+
+export const cacheProvider = (cache: Readonly<Cache<any>>) => ({
+  get: (key: string) => memoryCache.get(key),
+  set: (key: string, value: any) => memoryCache.set(key, value),
+  delete: (key: string) => memoryCache.delete(key),
+  keys: () => Array.from(memoryCache.keys())[Symbol.iterator]()
+});
+
+export const clearSWRCache = () => memoryCache.clear();
```

Comment:
based on https://swr.vercel.app/docs/advanced/cache

This might be a dangerous way to clear the cache

Reply:
Good catch, used the `mutate` strategy from the docs.